scripts/release.sh

```diff
@@ -3,34 +3,43 @@
 set -e
 
 source ${DAPPER_SOURCE}/scripts/lib/utils
+source ${SCRIPTS_DIR}/lib/debug_functions
+
+### Functions ###
+
+function determine_org() {
+    local origin_url=$(git config --get remote.origin.url)
+    basename $(dirname "$origin_url")
```

Comment:
This still doesn’t work for `git@` URLs — how about

    git config --get remote.origin.url | awk -F'[:/]' '{print $(NF-1)}'